lib/slim_lint/linter/trailing_blank_lines.rb

```diff
@@ -0,0 +1,18 @@
+module SlimLint
+  # Checks for trailing whitespace.
+  class Linter::TrailingBlankLines < Linter
+    include LinterRegistry
+
+    on_start do |_sexp|
+      dummy_node = Struct.new(:line)
+
+      if !document.source.end_with?("\n")
+        report_lint(dummy_node.new(document.source_lines.size),
+                    'No blank line in the end of file')
+      elsif document.source.lines.last.blank?
+        report_lint(dummy_node.new(document.source.lines.size),
+                    'Multiple empty lines in the end of file')
```

Comment:
Shouldn't this case be handled by the `EmptyLines` linter? Why are we duplicating the check?

In RuboCop, they have a `Style/FinalNewline` linter, which is what I think you want to emulate here, since you simply want to ensure whether or not the file has a trailing newline.